var/spack/repos/builtin/packages/mpc/package.py

```diff
@@ -27,15 +27,16 @@ class Mpc(AutotoolsPackage):
     depends_on('mpfr@3.0.0:', when='@1.1.0:')
 
     def url_for_version(self, version):
-        if version[-1] in string.ascii_letters:
-            version = version[0:-1].strip()
+        version_str = str(version)
+        if version_str[-1] in string.ascii_letters:
+            version_str = version_str[0:-1].strip()
```

Comment:
```suggestion
            version_str = version_str[:-1].strip()
```

Reply:
I'm not convinced that you really need the `strip`, too

Reply:
True. I don't like this workaround a lot though and I now saw that when `jenkins` builds `gcc` it uses the packages from `/gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/deploy/compilers/2020-02-01/linux-rhel7-x86_64/gcc-4.8.5/`, so redeploying a different version of `mpc` might not be needed to deploy `gcc 9.3.0`. The problematic `mpc` installation is the one included in `packages.yaml` and there might be an issue when building `gcc` locally

Reply:
If it's only GCC 9 that relies on `mpc`, then yes, it does not seem to make much sense to redeploy the one that ends up in `packages.yaml`, since that comes from tools/external-libraries.